src/views/SpikeAmplitudes/SelectUnitsWidget.tsx

```diff
@@ -0,0 +1,50 @@
+import NiceTable from 'components/NiceTable/NiceTable'
```

Comment:
This code is all pretty much what was in SortingView, right? (I haven't looked at it too closely, on the assumption that it's the code we had before.)

Reply:
yes, i pretty much copied this over